.idea/PytPOZ02_School.iml

```diff
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.7" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.7 (aDzbanuszek)" jdkType="Python SDK" />
```

Comment:
te pliki w .idea nie powinny byc zmieniane, a nawet jesli IDE wprowadzilo tam automatycznie zmiany to nie powinny być dodawane do projektu